commit 7b36f171d09354a2d3a48db0ae2d34d66aa4f1ae
Author: James Hilliard <james.hilliard1@gmail.com>
Date:   Sat Feb 1 20:02:29 2020 -0700

    unconditionally include sys/mman.h in os-compatibility.c
    
    Fixes:
     ../shared/os-compatibility.c:273:25: error: ‘PROT_READ’ undeclared (first use in this function); did you mean ‘LOCK_READ’?
       map = mmap(NULL, size, PROT_READ | PROT_WRITE, MAP_SHARED, file->fd, 0);
                              ^~~~~~~~~
                              LOCK_READ
    
    Signed-off-by: James Hilliard <james.hilliard1@gmail.com>

Upstream-Status: Backport

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

diff --git a/shared/os-compatibility.c b/shared/os-compatibility.c
index 5e1ce479..041c929f 100644
--- a/shared/os-compatibility.c
+++ b/shared/os-compatibility.c
@@ -34,10 +34,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <libweston/zalloc.h>
-
-#ifdef HAVE_MEMFD_CREATE
 #include <sys/mman.h>
-#endif
 
 #include "os-compatibility.h"
 
